### PR TITLE
Use eslint and prettier

### PR DESCRIPTION
**Pull request by smacker**
It will allow to use consistent code style

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,5 +30,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "prettier": {
```
Comment (dpordomingo) [inline package.json]: Is there no way to avoid this config here? Can not be moved any other place? :cry:

Comment (smacker) [reply package.json]: there are 3 options how to save config: https://prettier.io/docs/en/configuration.html
I prefer in package.json. But if you like .rc file, I'll change it

**Comment by dpordomingo** [comment]
@smacker you can merge once you had:
- separated the modifications in two commits:
  - first with the lint configurations and so,
  - the second one with all the modifications automatically made by the linter or required by it
- passed the @jbeardly requirement about the "Signed-off-by" [more info](https://github.com/src-d/guide/blob/master/engineering/licensing.md), and [here](https://github.com/bblfsh/dashboard/blob/master/CONTRIBUTING.md)

**Comment by smacker** [comment]
@dpordomingo rebased. I don't have rights to merge. You need to press a button.

**Comment by bzz** [comment]
LGTM. and @dpordomingo same lack of merge rights here

**Comment by smacker** [comment]
Yeah! I have right to merge now.
I rebased commits with sign off, but CI still says they are missed. It's weird. But looks like all good now.
I'm merging.

**Pull request merged**